zebrad/src/components/mempool/storage/verified_set.rs

```diff
@@ -286,10 +286,110 @@ impl VerifiedSet {
     }
 
     fn update_metrics(&mut self) {
+        // Track the sum of unpaid actions within each transaction (as they are subject to the
+        // unpaid action limit). Transactions that have weight >= 1 have no unpaid actions by
+        // definition.
+        let mut unpaid_actions_with_weight_lt20pct = 0;
+        let mut unpaid_actions_with_weight_lt40pct = 0;
+        let mut unpaid_actions_with_weight_lt60pct = 0;
+        let mut unpaid_actions_with_weight_lt80pct = 0;
+        let mut unpaid_actions_with_weight_lt1 = 0;
+
+        // Track the total number of paid actions across all transactions in the mempool. This
+        // added to the bucketed unpaid actions above is equal to the total number of conventional
+        // actions in the mempool.
+        let mut paid_actions = 0;
```

Comment:
```suggestion
        let mut paid_actions: i32 = 0;
```

Reply:
This should be the same type as conventional_actions and unpaid_actions. I think that's u32. (It definitely shouldn't be signed, negative values aren't valid.)